pages/dkp/kommander/2.1/install/air-gapped/catalog/index.md

```diff
@@ -110,3 +108,5 @@ To configure the Gitea server, follow these steps:
     ```
 
 1. After the newly created `GitRepository` on the management cluster reconciles, any corresponding `App`s are loaded by Kommander controller.
+
+This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0][https://www.gnu.org/licenses/agpl-3.0.en.html]. The complete source code for MinIO is available for the [2021 release][[https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z] of DKP/Kommander and the [2022 release][https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z] of DKP/Kommander.
```

Comment:
THought we were supposed to use Kevin's Kommander.txt files from google drive for these?
This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the GNU Affero General Public License 3.0. The complete source code for the versions of MinIO packaged with DKP/Kommander/Konvoy 2.1.1 are available at these URLs: 
https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z
https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z